conf/routes

```diff
@@ -48,6 +47,7 @@ GET         /collection/*collectionId                controllers.FaciaToolContro
 POST        /collections                             controllers.FaciaToolV2Controller.getCollections
 POST        /edits                                   controllers.FaciaToolController.collectionEdits
 POST        /v2Edits                                 controllers.FaciaToolV2Controller.collectionEdits
+POST        /collection/v2Discard/*collectionId      controllers.FaciaToolV2Controller.discardCollection(collectionId)
```

Comment:
It'd be really good to normalise these routes under a single /v2/:action path at some point!

Reply:
Yes, I agree, it's a bit of a mess